framework/core/locale/core.yml

```diff
@@ -661,6 +661,7 @@ core:
     # These translations are displayed as error messages.
     error:
       circular_dependencies_message: "Circular dependencies detected: {extensions}. Aborting. Please disable one of the extensions and try again."
+      db_error_message: "An error occurred while communicating with the database. This could mean an incompatibility with your database driver."
```

Comment:
```suggestion
      db_error_message: "Database query failed. This may be caused by an incompatibility between an extension and your database driver."
```